pagopa/io-app
# chore(IT Wallet): [SIW-1415] Add debug info to IT Wallet screens in debug mode

## Short description
This PR adds debug utilities and debug info in IT Wallet screens

## List of changes proposed in this pull request
- Added `withDebugEnabled` HOC to render components only when debug mode is enabled
- Refactored `DebugPrettyPrint` to allow copying to the clipboard its content by pressing on the title
- Added `DebugView` component, which displays its content only when debug mode is enabled with a nicer look
- Updated debug reducer to handle debug data
- Added redux actions to store/reset debug data
- Added `useDebugInfo` hook to display debug data in the debug overlay where necessay
- Added debug info in IT Wallet issuance screens

## How to test
Start an eID/mdl issuance flow , check that debug info overlay (little ladybug on top of the screen) is only visible when debug mode is enabled
